src/main/resources/db/migration/V1_82__create_delivery_session_tables.sql

```diff
@@ -0,0 +1,29 @@
+BEGIN;
+
+    drop view delivery_session;
```

Comment:
Seems to me that we should deploy these changes out of hours since DeliverySession entity is currently being used?

Perhaps it's not necessary, but also I don't know for sure.